boa3/model/builtin/native/nativecontract.py

```diff
@@ -28,9 +31,15 @@ class NativeContract:
                            types=[Policy]
                            )
 
+    RoleModule = Package(identifier=Role.identifier.lower(),
+                         types=[Role,
+                                RoleType]
```

Comment:
```suggestion
    RoleManagementModule = Package(identifier=Role.identifier.lower(),
                                   types=[RoleManagement,
                                          Interop.RoleType
                                          ]
```
for better documenting the code, use the full name of the implemented module